tools/code_format/check_format.py

```diff
@@ -888,10 +879,6 @@ def fix_source_path(self, file_path):
         if not file_path.endswith(PROTO_SUFFIX):
             error_messages += self.fix_header_order(file_path)
         error_messages += self.clang_format(file_path)
-        if file_path.endswith(PROTO_SUFFIX) and self.is_api_file(file_path):
-            package_name, error_message = self.package_name_for_proto(file_path)
-            if package_name is None:
```

Comment:
these lines are also wrong - it just adds the same error message a second time, incorrectly